### PR TITLE
Complete Login page

### PR DESCRIPTION
**Pull request by ValeryiaMIRON**
- Add footer
- Finish Login Page
Снимок экрана 2021-10-20 в 9 28 56 PM

https://trello.com/c/I5ez3pOS/55-fe-complete-page-login

### CODE REVIEW

File: src/pages/login/Login.jsx [hunk 1]
```diff
@@ -12,14 +11,32 @@ class Login extends React.Component {
...
     return (
-      <Card className="loginCard">
-        <Typography m="15px" variant="h3" textAlign="center">Log in</Typography>
-        <Stack m="20px auto" width="350px" height="200px" spacing={2} direction="column">
-          <TextField id="outlined-basic" label="E-mail" variant="outlined" />
-          <TextField id="outlined-password-input" label="Password" type="password" autoComplete="current-password" />
-          <Button variant="contained">Log In</Button>
-        </Stack>
-      </Card>
+      <Box className="login-container">
```
Comment (nsukhotskaya) [inline src/pages/login/Login.jsx:14]: Classes names and ides have to be written in camelCase

File: src/App.scss [hunk 2]
```diff
@@ -1,7 +1,4 @@
-@import "colors";
-
-.App {
-  width: 100%;
-  height: 100vh;
-  background: $common-background;
+* {
+  padding: 0;
+  margin: 0;
 }
```
Comment (itsnotpolza) [inline src/App.scss:4]: you sure you want to use a * selector? It's kinda dangerous but i won't mind it if @nsukhotskaya doesn't mind

Comment (nsukhotskaya) [reply src/App.scss:4]: I agree with you. Better to avoid this approach.

File: src/pages/login/Login.jsx (hunk 1 above)
Comment (ValeryiaMIRON) [reply src/pages/login/Login.jsx]: done

**Review by nsukhotskaya — APPROVED** [review]

**Review by beliavskaja — APPROVED** [review]

**Review by oleksiuk-ol — APPROVED** [review]

**Pull request merged**